packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.component.js

```diff
@@ -54,6 +56,12 @@ export default class MultiSelectTag extends React.Component {
 		}
 	}
 
+	getTitleMap(props) {
+		return (
+			this.state.titleMap || (props && props.schema.titleMap) || this.props.schema.titleMap || []
```

Comment:
IMHO (props && props.schema.titleMap) is more safe with lodash get(props, 'schema.titleMap')